_data/AniruddhaTapas/AniruddhaTapas.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Aniruddha Tapas
+institution: Northeastern University
+profile_pic: aniruddha.png
+quote: Life may be tough at times, but destiny isn't created by the shoes we wear but by the steps we take.
```

Comment:
the quote has an exact 100 char, great job!

Reply:
Thanks! 😄